package.json

```diff
@@ -10,34 +10,32 @@
     },
     "scripts": {
         "analyze": "lit-analyzer \"{packages,tools}/*/src/**/!(*.css).ts\"",
+        "analyze:docs": "cem analyze --globs \"packages/**/*.ts\" --exclude \"**/*.d.ts\" --exclude \"**/stories/**\" --exclude \"**/icons/**\" --exclude \"**/elements/**\" --outdir projects/documentation --litelement",
+        "analyze:storybook": "wireit",
         "build": "wireit",
-        "build:clear-cache": "rimraf packages/*/tsconfig.tsbuildinfo && rimraf tools/*/tsconfig.tsbuildinfo",
-        "build:component-inventory": "node ./tasks/build-component-inventory.js",
+        "build:clean": "rimraf packages/*/tsconfig.tsbuildinfo tools/*/tsconfig.tsbuildinfo",
         "build:confirm": "node ./tasks/confirm-build.js",
         "build:css": "wireit",
         "build:css:watch": "wireit",
-        "build:react": "yarn gen-react-wrapper && node ./tasks/build-react.js && yarn tsc --build tsconfig-react-wrapper.json",
+        "build:icons": "wireit",
+        "build:react": "node ./tasks/build-react.js && tsc --build tsconfig-react-wrapper.json",
         "build:tests": "tsc --build test/tsconfig.json && tsc --build test/tsconfig-node.json",
         "build:ts": "wireit",
         "build:ts:watch": "wireit",
         "build:types": "wireit",
         "build:watch": "wireit",
-        "changeset-snapshot-publish": "yarn prepublishOnly && yarn changeset version --snapshot && yarn lint:versions --fix && yarn update-version && yarn changeset publish --no-git-tag --tag snapshot",
-        "changeset-publish": "yarn prepublishOnly && yarn changeset version && yarn install && yarn lint:versions --fix && yarn update-version && yarn changeset publish --no-git-tag && yarn push-to-remote && yarn create-git-tag && yarn postpublish",
-        "update-version": "node ./tasks/update-version.js",
+        "changeset-publish": "yarn prepublishOnly && yarn changeset version && yarn install && yarn lint:versions --fix && yarn update-version && yarn changeset publish && yarn postpublish",
+        "changeset-snapshot-publish": "yarn prepublishOnly && yarn changeset version --snapshot && yarn install && yarn lint:versions --fix && yarn update-version && yarn changeset publish --no-git-tag --tag snapshot",
```

Comment:
These are the same, just reordered alphabetically.

Reply:
`changeset-publish` will create Github tags.

`changeset-snapshot-publish` does not create Github tags as expected with snapshot releases since we don't maintain them. 

I hope this helps provide clarity, let me know if you still have Qs

Reply:
OH i see what youre referencing in the diff...... hmmm good questions

Reply:
I did a ton of digging and research on this so we can chat this morning